core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStoreReaderThread.java

```diff
@@ -73,7 +76,18 @@ public KafkaStoreReaderThread(String bootstrapBrokers,
                                 StoreUpdateHandler<K, V> storeUpdateHandler,
                                 Serializer<K, V> serializer,
                                 Store<K, V> localStore,
-                                K noopKey) {
+                                K noopKey,
+                                String kafkastoreSecurityProtocol,
```

Comment:
Hmm, not sure about this approach to passing in the configs -- seems like an excessive number of arguments. Any reason we can't just pass around the original configs map and pull out exactly the set of configs we need? The current approach seems like it could get especially messy if we add support for anything besides SSL.